setup.py

```diff
@@ -91,16 +98,23 @@ def run(self):
         ]
     },
     license="Apache License 2.0",
-    long_description=readme,
+    long_description=package_desc,
     long_description_content_type="text/markdown",
     keywords="taipy",
     name="taipy",
     packages=find_packages(include=["taipy", "taipy.*"]),
     include_package_data=True,
     test_suite="tests",
-    url="https://github.com/avaiga/taipy",
     version=version_string,
     zip_safe=False,
     extras_require=extras_require,
     cmdclass={"build_py": NPMInstall},
+    project_urls={
+        "Homepage": "https://www.taipy.io",
+        "Documentation": "https://docs.taipy.io",
+        "Source": "https://github.com/Avaiga/taipy",
+        "Download": "https://pypi.org/project/taipy/#files",
+        "Tracker": "https://github.com/Avaiga/taipy/issues",
+        "Release notes": "https://docs.taipy.io/en/latest/relnotes/",
```

Comment:
Please add the [security policy](https://github.com/Avaiga/taipy?tab=security-ov-file#readme)

Reply:
Should we use variables in the documentation and release notes links? If the user is not on the latest version of the product, he won't want to go to the latest version.

Reply:
That would be nice, yes. Let me do that